.gitignore

```diff
@@ -8,3 +8,4 @@ target/
 /tests/fixtures/repos
 
 **/generated/
+.idea
```

Comment:
This belongs into `~/.gitignore`.

Reply:
Forgot: https://www.toptal.com/developers/gitignore - a nice generator for all kinds of standard .gitignore files.